roadmap.md

```diff
@@ -102,7 +102,7 @@ widely-used batch processors, there are some ongoing efforts to make sure this t
     intermediate results for better resource efficiency on systems like Yarn.
     [FLIP-31](https://cwiki.apache.org/confluence/display/FLINK/FLIP-31%3A+Pluggable+Shuffle+Manager).
 
-Various of these enhancements can be taken form the contributed code from the
+Various of these enhancements can be taken from the contributed code from the
```

Comment:
hi, I think _roadmap.zh.md_ should also apply this change.

Reply:
Fixed it too, Thank you.